abs.yaml

```
@@ -1,4 +1,5 @@
 # Required for glibc >= 2.26 for intel-openmp to work.
 pkg_build_image_tag: main-rockylinux-8
 build_env_vars:
-  ANACONDA_ROCKET_GLIBC: "2.28"
\ No newline at end of file
+  ANACONDA_ROCKET_GLIBC: "2.28"
+  ANACONDA_ROCKET_ENABLE_PY314 : yes
```

rocky8 and GLIBC 2.28 are the defaults, now:


```suggestion
build_env_vars:
  ANACONDA_ROCKET_ENABLE_PY314 : yes
```